### PR TITLE
https://issues.redhat.com/browse/ACM-6806 Add decisionStrategy Placement

### PR DESCRIPTION
**Pull request by oafischer**
2.10 only

### CODE REVIEW

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 1]
```diff
@@ -34,7 +33,94 @@ status:
...
+    decisionGroupIndex: 3
+    decisionGroupName: ""
+    decisions:
+    - placement1-decision-5
+    - placement1-decision-6
+  numberOfSelectedClusters: 310
+----
+
+The `PlacementDecision` has the `cluster.open-cluster-management.io/decision-group-name` 
+and `cluster.open-cluster-management.io/decision-group-index` labels to indicate which grooup name and group index it belongs to. See the following example:
```
Comment (hchenxa) [inline clusters/cluster_lifecycle/placement_decision.adoc]: typo , grooup should be group.

Comment (oafischer) [reply clusters/cluster_lifecycle/placement_decision.adoc]: Good catch, fixed!

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 2]
```diff
@@ -34,7 +33,94 @@ status:
       reason: ''
 ----
 
+[#placement-decisionstrategy]
+== Dividing _PlacementDecision_ into groups by using _decisionStrategy_
+
+You can use the `decisionStrategy` spec in a `Placement` to divide the `PlacementDecision` you created into groups and define the number of clusters in each group.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_decision.adoc]: I recommend writing out spec or you can monospace it : `spec`
```suggestion
You can use the `decisionStrategy` specification in a `Placement` to divide the `PlacementDecision` resource that you created into groups and define the number of clusters in each group.
```

File: clusters/cluster_lifecycle/placement_decision.adoc [hunk 3]
```diff
@@ -34,7 +33,94 @@ status:
...
+              - key: prod-canary-east
+                operator: Exists
+----
+
+<1> A specific number or percentage of the total selected clusters 
+<2> A list of predefined groups for adding decision results. Decision groups are initially made based on the `decisionGroups` field. Clusters that are not included in `decisionGroups` are then divided to other decision groups. The number of cluster in each decision group must be lower than the value of `clustersPerDecisionGroup`.
+<3> The name that is added as the value of the `cluster.open-cluster-management.io/decision-group-name` label key of the `PlacementDecision` you created.
+<4> Defines the `labelSelector` to select a cluster subset by using a label.
+
+After defining the decision strategy, the `Placement` `status` lists the group result, including the decision group name and index, cluster count, and the corresponding `PlacementDecision` names. See the following example:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/placement_decision.adoc]:
```suggestion
After you define the decision strategy, the `Placement` `status` lists the group result, including the decision group name and index, cluster count, and the corresponding `PlacementDecision` names. See the following example:
```

File: clusters/release_notes/whats_new.adoc [hunk 4]
```diff
@@ -16,23 +16,8 @@ Learn more about what is new this release:
  
 Learn about what's new relating to Cluster lifecycle with {mce-short}.
 
-Use the integrated console to create a credential and cluster that runs in the Nutanix environment. See link:../credentials/credential_intro.adoc#credentials[Managing credentials] and link:../cluster_lifecycle/create_intro.adoc#create-intro[Cluster creation introduction] for more information. 
-
+You can now use the `decisionStrategy` spec in a `Placement` to divide a `PlacementDecision` into groups and define the number of clusters in each group. See xref:../cluster_lifecycle/placement_decision.adoc#placement-decisionstrategy[Dividing _PlacementDecision_ into groups by using _decisionStrategy_]
```
Comment (dockerymick) [inline clusters/release_notes/whats_new.adoc]: same suggestion

**Review by dockerymick — APPROVED** [review]
Made some suggestions

**Pull request closed without merging**